File: src/main/java/com/datastax/cdm/NewCommand.java
package com.datastax.cdm;

import com.beust.jcommander.Parameter;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by jhaddad on 10/7/16.
 */
public class NewCommand {
    @Parameter
    public List<String> dataset = new ArrayList<>();
}

File: src/main/kotlin/com/rustyrazorblade/cdm/NewCommand.kt
package com.rustyrazorblade.cdm

import com.beust.jcommander.Parameter
import com.beust.jcommander.Parameters

import java.util.ArrayList

/**
 * Created by jhaddad on 10/7/16.
 */
@Parameters(commandDescription = "New Dataset")
class NewCommand {
    @Parameter
    var dataset: List<String> = ArrayList()
}
